### PR TITLE
Phase2-hgx359Y Update the scripts in Alignment/OfflineValidation and Geometry/HcalCommonData|MTDCommonData for the changes in Configuration/Geometry

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Update the scripts in Alignment/OfflineValidation and Geometry/HcalCommonData|MTDCommonData for the changes in Configuration/Geometry

#### PR validation:

Tested the modified scripts

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by perrotta** [comment]
+alca

**Comment by civanch** [comment]
+1

**Comment by srimanob** [comment]
+Upgrade

File: Alignment/OfflineValidation/test/test_all_Phase2_cfg.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ class RefitType(Enum):
 # Standard loads
 ###################################################################
 #process.load("Configuration.Geometry.GeometryRecoDB_cff")
-process.load('Configuration.Geometry.GeometryExtended2026D88Reco_cff')
+process.load('Configuration.Geometry.GeometryExtended2026D110Reco_cff')
```
[the remark this answers is not in the document]
Comment (perrotta) [reply Alignment/OfflineValidation/test/test_all_Phase2_cfg.py:84]: `auto:phase2_realisitc` is as of now in master `141X_mcRun4_realistic_v1`, which is configured for the D110 geometry (phase2_realistic_T33) 
I would say it is in synch

[the remark this answers is not in the document]
Comment (mmusich) [reply Alignment/OfflineValidation/test/test_all_Phase2_cfg.py:84]: perfect thanks for the check.

**Comment by mandrenguyen** [comment]
+1

**Pull request merged**